api/base/filters.py

```diff
@@ -430,6 +430,8 @@ def postprocess_query_param(self, key, field_name, operation):
                 else self.model_class.primary_identifier_name
             )
             operation['op'] = 'in'
+            if not isinstance(operation['value'], list):
```

Comment:
I'm not sure this belongs here. The parsing of `filter[id]=123,abc` happens in `bulk_get_values`. Perhaps that method should handle when single values are passed.